baseband/vlbi_base/header.py

```diff
@@ -467,15 +467,15 @@ def __setitem__(self, item, value):
             else:
                 raise
 
-    def __getattr__(self, attr):
-        """Get attribute, or, failing that, try to get key from header."""
-        try:
-            return super(VLBIHeaderBase, self).__getattribute__(attr)
-        except AttributeError:
-            if attr in self.keys():
-                return self[attr]
-            else:
-                raise
+    # def __getattr__(self, attr):
```

Comment:
Please just remove it altogether!

Reply:
Obviously I need to get into the habit of doing `git diff HEAD~1` before I submit PRs...